geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -245,8 +268,17 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         goog.asserts.assertInstanceof(evt.element, ol.Feature);
         var feature = evt.element;
-        goog.events.unlisten(feature, goog.events.EventType.CHANGE,
-            this.onFeatureChange_, undefined, this);
+        if (goog.isDefAndNotNull(feature.get('__source__')) &&
+            feature.get('__source__') == 'mymaps') {
+          var map_id = this.appMymaps_.getCurrentMapId();
```

Comment:
For consistency reasons I'd recommend not to use the underscore notation for js variables.